### PR TITLE
Avoid using long prefixed names for endpoint payloads

### PR DESCRIPTION
**Pull request by e1himself**
Followup on #163 

See https://github.com/prezly/javascript-sdk/pull/163#discussion_r1011867456

### CODE REVIEW

File: src/endpoints/NotificationSubscriptions/types.ts [hunk 1]
```diff
@@ -1,6 +1,12 @@
...
         notification_type: string;
         is_email_subscribed: boolean;
     }[];
 }
+
+/**
+ * @deprecated This is a BC alias to `UpdateRequest`. Please use `UpdateRequest` directly instead.
+ * @see UpdateRequest
+ */
+export type NotificationSubscriptionsUpdateRequest = UpdateRequest;
```
[the remark this answers is not in the document]
Comment (oleg-semyonov) [reply src/endpoints/NotificationSubscriptions/types.ts:12]: Nevermind, I saw that it was already released as prod version

**Review by oleg-semyonov — APPROVED** [review]

**Comment by oleg-semyonov** [comment]
Hey @marko-ogg @e1himself, I noticed that Notification subscription types don't use namespaces, so that might as well be addressed in this PR.

That would also resolve the issue that `UpdateRequest` type is currently not diretcly exported from the package.

**Comment by e1himself** [comment]
> That would also resolve the issue that UpdateRequest type is currently not diretcly exported from the package.

What do you mean @oleg-semyonov? It's exported as a part of the `NewsroomSubscriptions` namespace.

https://github.com/prezly/javascript-sdk/blob/c3271ce65a876ef6f2a404dbb7b40a41056f27ff/src/endpoints/index.ts#L21

**Comment by oleg-semyonov** [comment]
Oh sorry, I was looking at the wrong thing. Disregard my comment.

File: src/endpoints/NotificationSubscriptions/types.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (e1himself) [reply src/endpoints/NotificationSubscriptions/types.ts]: I'll just bump it to v11. Too many things wrong :) 

Package-scoped `NotificationType` is confusing too. As it looks like it's related to the package-scoped `Notification` interface. But it's not.

**Review by oleg-semyonov — APPROVED** [review]

**Review by marko-ogg — APPROVED** [review]
Thanks @e1himself for putting this in place! 🙏🏻

**Pull request merged**